apps/src/applab/applab.js

```diff
@@ -422,14 +416,32 @@ Applab.init = function (config) {
     }));
   }
   Applab.channelId = config.channel;
-  Applab.storage = initFirebaseStorage({
-    channelId: config.channel,
-    firebaseName: config.firebaseName,
-    firebaseAuthToken: config.firebaseAuthToken,
-    firebaseSharedAuthToken: config.firebaseSharedAuthToken,
-    firebaseChannelIdSuffix: config.firebaseChannelIdSuffix || '',
-    showRateLimitAlert: studioApp().showRateLimitAlert,
-  });
+
+  // TODO: post-firebase-cleanup, remove this conditional when we're removing firebase: #56994
+  if (!!config.useDatablockStorage) {
+    // TODO: post-firebase-cleanup, remove this console.log: #56994
```

Comment:
Why are the console logs needed at all?

Reply:
Might be useful to have in a console screenshot if we get zendesk tickets, but we can also look it up for a given project. We were thinking of removing them anyway since they're kinda noisy.

Reply:
We chatted, and decided it'd be better to give Hannah a quick way to check if a given URL is on datablock storage or firebase storage, so removing console.logs